setting-up-multifactor-authentication.md

```diff
@@ -3,89 +3,14 @@ layout: default
 title: Setting up multi-factor authentication
 url: /setting-up-multifactor-authentication
 previous: /run-your-own-gem-server
-next: /using-mfa-in-command-line
+next: /setting-up-webauthn-mfa
 ---
 
 <em class="t-gray">Want to better protect your rubygems.org account?</em>
 
 Your rubygems.org account is important! Unauthorized access of your account
 can lead to irrevocable damage to your gem's reputation. We highly recommend
-that you enable MFA for both UI and API. When enabled, we will ask you to provide a
-one-time password for operations like login, gem push etc to verify your
-identity.
+that you enable MFA for both UI and API.
```

Comment:
I'd like to see us keep this explanation of what operations you might use MFA for. I think it provides helpful context

Reply:
Good thinking. IIRC it's triggered for `signin`, `owner` and `push`. Is that right?

Reply:
Also `yank`